cargo-sqlx/src/postgres.rs

```diff
@@ -89,7 +89,7 @@ impl DatabaseMigrator for Postgres {
 
         let mut conn = PgConnection::connect(base_url).await?;
 
-        sqlx::query(&format!("CREATE DATABASE {}", db_name))
+        sqlx::query(&format!("CREATE DATABASE \"{}\"", db_name))
```

Comment:
For sanity, let's forbid double-quotes from appearing in the database name.

Reply:
We should probably add a general purpose `quote` function that is db-specific. Escaping double quotes is done by doubling the quotes in pg.

Reply:
`db_name.replace('"', "\"\"")`

Reply:
Pushed up a fix to allow quotation marks in db name. I do agree that using it should be discouraged.